DevoxxClientMobile/src/main/resources/com/devoxx/views/badges.fxml

```diff
@@ -1,5 +1,22 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
 <?import com.gluonhq.charm.glisten.mvc.View?>
-<View fx:id="badgesView" xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1" fx:controller="com.devoxx.views.BadgesPresenter">
+<?import javafx.scene.layout.VBox?>
+<?import javafx.scene.control.Button?>
+<?import com.gluonhq.charm.glisten.control.Icon?>
+<View fx:id="badgesView" styleClass="badges-view" xmlns="http://javafx.com/javafx/8" xmlns:fx="http://javafx.com/fxml/1" fx:controller="com.devoxx.views.BadgesPresenter">
+    <center>
+        <VBox fx:id="content" styleClass="content">
+            <Button fx:id="sponsor" text="Sponsor" onAction="#showSponsor">
+                <graphic>
+                    <Icon content="FOLDER_SPECIAL"/>
```

Comment:
I believe CARD_GIFTCARD and EVENT_SEAT or better icons. And can we at the same time use the same icon for both sponsor views as well (currently defined as PAYMENT in DevoxxView).

Reply:
EVENT_SEAT looks nice for attendee. For sponsor, CARD_MEMBERSHIP looks better than CARD_GIFTCARD?

Reply:
CARD_MEMBERSHIP is fine with me too.